### DISCUSSION TITLE
Stylus Touch Issue after upgrade to 106.0.29

### DISCUSSION THREAD

**Discussion by bmck11** [cefsharp/CefSharp#4397] [General]
Hello,

After finally upgrading CefSharp from 73.1.13 to 106.0.29, it is noticed that when we use the CefSharp app alongside TinyMCE to make a text editor (add text and control it..ex: bold, italic)...that on a Windows10 Panasonic CF-20, the stylus touch is no longer working in this text editor component.  The finger presses work and the mousepad clicks work.  I cannot click at all with the stylus.  I even upgraded CEF to 110.0.25 and this issue still occurs.  I installed the older version of our app that has CEF 73.1.13 and it works like a dream.

Please advise and I already searched (sadly) high and low).

**What version of the product are you using?**
106.0.29 and 110.0.25

**What version of .Net?**
.Net 4.5.2

**On what operating system?**
Win10

**Are you using WinForms, WPF or OffScreen?**
WPF

What steps will reproduce the problem?
Attempt to use stylus touch in text editor

**What is the expected output? What do you see instead?**
There is no click and cursor is not inserted into the line of text as expected.  Works fine if you use finger touch or mousepad touch to click into text editor and go to that exact spot.

Thanks,

Team Stylus

**Comment by amaitland** [comment]
See https://github.com/cefsharp/CefSharp/issues/3028

> I installed the older version of our app that has CEF 73.1.13 and it works like a dream.

This is likely before touch support was added, best guess is that it is emulating the mouse. If the newer stylus implementation doesn't work as expected then you can probably override the mouse handling to obtain the old behaviour. 

Try the experimental implementation first.

**Reply by bmck11** [reply]
@amaitland ,

I like your idea about "override the mouse handling to obtain the old behaviour" do you know where in the code I can override the mouse new behavior, or how to look for that code ?

**Reply by amaitland** [reply]
There are checks for `StylusDevice == null` in the code, I'd be starting there.
